### ISSUE TITLE
ColorPalettePreference is not full height in version 4.0.0

### ISSUE THREAD

**Issue by Kahtaf** [michael-rapp/AndroidMaterialPreferences#19]
Hi Michael, thanks for this awesome library. I noticed an issue after upgrading to v4.0.0, the `ColorPalettePreference` is not full height anymore. I didn't change anything with the theme or layout, only the changes required to migrate the library from the older version. 

Below is a screenshot of what it looks like currently, whereas previously, the dialog was full height (similar to the `ColorPalettePreference` screenshot in your readme). There are about 50 colors in this dialog (notice the tiny scrollbar), so it's not an issue with it being only 3 colors. 

Here is the relevant code. I tried setting a `dialogHeight` and `dialogMaxHeight`, but those didn't take any affect. 

```
        <de.mrapp.android.preference.ColorPalettePreference
            android:defaultValue="@color/bottom_bar_background"
            android:key="pref_bottom_nav_color"
            android:title="@string/pref_bottom_nav_color"
            custom:colorFormat="hex4bytes"
            custom:colorPalette="@array/color_picker_options"
            custom:dialogHeight="400dp"
            custom:dialogMarginBottom="10dp"
            custom:dialogMarginLeft="10dp"
            custom:dialogMarginRight="10dp"
            custom:dialogMarginTop="10dp"
            custom:dialogMaxHeight="400dp"
            custom:showPreview="true"
            custom:showValueAsSummary="false" />
```

Is it an issue with the library or did I miss something? Thank you.

![screenshot_automate_20180607-094517](https://user-images.githubusercontent.com/5796181/41103888-388d76c8-6a38-11e8-8e58-c6628b965d6a.png)

**Comment by michael-rapp** [comment]
Thank you for reporting this issue. It is in fact a bug. I don't think that you have missed something. In version 4.0.0 the library "AndroidMaterialDialog" this library relies on was updated, which introduced significant changes in how scrollable content is handled.

**Comment by michael-rapp** [comment]
Fixed in version 5.0.0.
